runtime/compiler/env/J9SharedCache.cpp

```diff
@@ -936,10 +962,18 @@ TR_J9SharedCache::rememberClass(J9Class *clazz, bool create)
    if (chainData)
       {
       LOG(1, "\tstored data, chain at %p\n", chainData);
+      if (comp)
+         {
+         ADD_TRACING_BUFFER_MESSAGE(comp, "rememberClass: stored data, chain at %p, %p, %p\n", chainData, clazz, romClass);
+         }
```

Comment:
It appears that all uses of `ADD_TRACING_BUFFER_MESSAGE` are guarded by `if (comp)`. `ADD_TRACING_BUFFER_MESSAGE` accepts `comp` as the first argument. Would it be possible to inline this check into the `#define` and save ourselves 3 lines per usage?

Reply:
There's actually several uses in `runtime/compiler/env/j9method.cpp` that don't have the `if (comp)`. My intention was to only do the check when absolutely necessary because this code may be timing sensitive.

Reply:
Right, I see it now. Thanks.